examples/seccompprofile.yaml

```diff
@@ -2,23 +2,101 @@
 apiVersion: security-profiles-operator.x-k8s.io/v1alpha1
 kind: SeccompProfile
 metadata:
-  name: profile-block
+  name: profile-block-all
```

Comment:
Might be possible that we have to adapt the test-cases for this change:
https://github.com/kubernetes-sigs/security-profiles-operator/blob/5580b9f684c8dc267ba78de5647f9330f4c629af/test/tc_default_profiles_test.go#L32

Reply:
Yep, went across the project and updated other references to it in documentation/tests. PTAL